action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Appreciation Engine whitelist'
+description: 'Whitelists or unwhitelists domains for appreciation engine'
+inputs:
+  operation: 
+    description: 'whitelist or unwhitelist'
+    required: true
+    default: 'whitelist'
+  apiKey:
+    description: 'Appreciation Engine API key'
+    required: true
+    default: ''
+  domain: 
+    description: 'Domain for (un-)whitelisting'
+    required: true
+    default: 'example.com'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        curl -X POST \
+        --data "apiKey=${{ inputs.apiKey }}" \
+        --data "domain=${{ inputs.domain }}" \
+        https://staging-sme.theappreciationengine.com/v1.1/app/${{ inputs.operation }}?
```

Comment:
Do we need this configrable? Maybe useful for others who need to affect production environments not staging?

Can leave for this initial version.